web/app/package.json

```diff
@@ -30,8 +30,8 @@
     "sinon": "^4.1.2",
     "sinon-chai": "^2.14.0",
     "sinon-stub-promise": "^4.0.0",
-    "webpack": "^4.1.12",
-    "webpack-dev-server": "^3.1.4",
+    "webpack": "^4.16.5",
```

Comment:
More of a nit, I would've preferred for this to be in separate PR, but the change is so small enough that I think its fine.